auth-web/src/util/debounce.ts

```diff
@@ -0,0 +1,9 @@
+export function debounce<F extends (...params: any[]) => void>(fn: F, delay = 300) {
+  let timeoutID: number = null
+  return function (this: any, ...args: any[]) {
```

Comment:
can we use the same debounce function in fas-ui
https://github.com/bcgov/fas-ui/blob/19542cd7ea68b5b8a89557cf8496ddad5ee576d5/src/util/debounce.ts

Reply:
I tried that, unfortunately it doesn't pass the code quality.

Reply:
Not sure what you mean by doesn't pass code quality